docs/reference/commandline/export.md

```diff
@@ -12,19 +12,17 @@ weight=1
 # export
 
     Usage: docker export [OPTIONS] CONTAINER
-    
-      Export the contents of a filesystem to a tar archive (streamed to STDOUT by default).
 
-      -o, --output=""    Write to a file, instead of STDOUT
-
-      Produces a tarred repository to the standard output stream.
+    Export the contents of a container's filesystem as a tar archive
 
+      --help=false       Print usage
+      -o, --output=""    Write to a file, instead of STDOUT
 
- For example:
+## Examples
```

Comment:
The examples section should be after the description:

...on exporting data in a volume.

## Examples